lib/matplotlib/backends/backend_macosx.py

```diff
@@ -182,12 +184,7 @@ class _BackendMac(_Backend):
 
     @staticmethod
     def trigger_manager_draw(manager):
-        # For performance reasons, we don't want to redraw the figure after
```

Comment:
This comment is true, but is not specific to the osx backend -- all interactive backends do the same.